src/test/java/seedu/lifeasier/storage/FileCommandTest.java

```diff
@@ -0,0 +1,32 @@
+package seedu.lifeasier.storage;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class FileCommandTest {
+
+    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("dd-MM-yy HH:mm");
```

Comment:
Could consider using the DATE_TIME_FORMATTER from the FileCommand class.
There's no need for constants to be private, and if we ever decide to change the DATE_TIME_FORMATTER there would be less stuff to change.

Reply:
Alright, updated the code accordingly